k8s/device-monitor/values-prod.yaml

```diff
@@ -1,35 +1,41 @@
-# values for airqo-device-registry-api.
-env:
-  deployment_name: airqo-device-monitor-api
-  service_name: airqo-device-monitor-api-svc
-  labels: dev-monitor-api
-  secret: env-device-monitoring-production
+app:
+  name: airqo-device-monitor-api
+  label: device-monitor
+  namespace: production
```

Comment:
Hi @123MwanjeMike , what is this `device-monitor` microservice about?

Reply:
I don't know for for sure @Baalmart 
I assumed that you would be the best person to know about it.
Perhaps it could be a deletion candidate.